tools/explorer/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(TT_EXPLORER_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/run.py)
+set(TTMLIR_BUILD_BIN_DIR ${TTMLIR_BINARY_DIR}/bin)
+
+add_custom_target(explorer
+  COMMENT "Building tt-explorer... ${TTMLIR_BIN_DIR}"
+  COMMAND cp -r ${CMAKE_CURRENT_SOURCE_DIR}/* ${CMAKE_CURRENT_BINARY_DIR}
+  COMMAND pip install ${CMAKE_CURRENT_BINARY_DIR}/tt_adapter
+  DEPENDS TTMLIRPythonModules
+)
+
+add_custom_command(TARGET explorer POST_BUILD
+  COMMENT "Installing tt-explorer command..."
+  COMMAND ${CMAKE_COMMAND} -E copy ${TT_EXPLORER_SCRIPT} ${TTMLIR_BUILD_BIN_DIR}/tt-explorer
+  COMMAND ${CMAKE_COMMAND} -E echo "Done. Run: tt-explorer to start the server."
+)
```

Comment:
This might be better off if it was added to the `pyproject.toml` as a script for the explorer tool. Check out [pyproject.toml](https://github.com/vprajapati-tt/tt-explorer/blob/main/pyproject.toml):20 for an example.

Reply:
I have considered using a pyproject.toml file but decided against it because the only feature we need from it is to make an executable. We don't need to have explorer be a python installable module or have any dependencies atm, so it seemed like overkill to have it.

We do have the pyproject.toml in tt_adapter/ but the run script has a bigger scope then just the adatper.

Reply:
Yeah that sounds good :)